src/gui_common/TransitionManager.cs

```diff
@@ -1,167 +1,201 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using Godot;
 
 /// <summary>
 ///   Manages the screen transitions, usually used for when
 ///   switching scenes. This is autoloaded
 /// </summary>
-public class TransitionManager : NodeWithInput
+public class TransitionManager : ControlWithInput
 {
-    private static TransitionManager? instance;
+    private static readonly PackedScene ScreenFadeScene = GD.Load<PackedScene>("res://src/gui_common/ScreenFade.tscn");
+    private static readonly PackedScene CutsceneScene = GD.Load<PackedScene>("res://src/gui_common/Cutscene.tscn");
```

Comment:
I think it would be better to load these in _Ready or the constructor like was done before, I like avoiding Godot resource loading that happens on assembly load time.

Reply:
Static readonlies can only be initialized this way or in a static constructor.

Reply:
I'd make this an instance specified readonly field then, as that's why preferred way here.